pkg/api/grids/grids_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package grids
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/controller"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutilcontroller"
+	"github.com/verrazzano/verrazzano-operator/pkg/types"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+var modelBindingPairs = map[string]*types.ModelBindingPair{
+	"test-pair-1": testutil.GetModelBindingPair(),
+}
+
+//  TestReturnAllGrids verifies that the single test-coherence grid entry is returned and valid
```

Comment:
I think the comments are suppose to follow what Mark has proposed. For example,
// TestCreateVMIServiceAccounts tests creation of service accounts when the binding name is 'system'
// GIVEN clusters which do not have service accounts for each namespace of a VMI binding
//  WHEN I call CreateServiceAccounts
//  THEN there should be a service account created for each namespace of a VMI binding
//   AND that service account should have the expected values

Reply:
Done